code/edgebridge/src/main/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtension.java

```diff
@@ -149,18 +149,18 @@ void handleRulesEngineResponse(@NonNull final Event event) {
 			return;
 		}
 
-		dispatchTrackRequest(detail, event.getTimestamp());
+		dispatchTrackRequest(detail, event);
 	}
 
 	/**
 	 * Helper to create and dispatch an experience event.
 	 * @param data map containing free-form data to send to Edge Network
-	 * @param timestamp timestamp of Event
+	 * @param parentEvent the parent event that triggered this request
```

Comment:
can we update description here, e.g.
@param parentEvent the triggering parent event used for event chaining; its timestamp is set as xdm.timestamp